api/v2/serializers/post/token_update.py

```diff
@@ -54,14 +54,14 @@ def create(self, data):
         endpoint_catalog = catalog['token']['catalog']
         nova = {}
         for l in endpoint_catalog:
-            if l['name'] == 'nova':
-                nova = l
+            if l['type'] == 'compute':
+                compute = l
```

Comment:
This doesn't match the name of the variable defined above anymore. Neither does compute_url match nova_url.